### PR TITLE
Fix EZP-23693: Http Headers are not set when using FastCGI

### PR DESCRIPTION
**Pull request by blankse**
JIRA: https://jira.ez.no/browse/EZP-23693

### CODE REVIEW

**Comment by blankse** [comment]
ping @bdunogier @dpobel

**Comment by andrerom** [comment]
@blankse as asked in the issue, any idea why this was not detected before? fast cgi has been used by some for years, and fpm for some years now as well. Is this for a specific setup or fpm version?

File: lib/ezutils/classes/ezhttptool.php [hunk 1]
```diff
@@ -615,17 +609,31 @@ static function redirect( $path, $parameters = array(), $status = false, $encode
...
+
+    /**
+     * Sets the HTTP response code
+     *
+     * @param int $httpResponseCode
+     */
+    static function sendHTTPResponseCode( $httpResponseCode )
     {
-        header( $headerName .': '. $headerData );
+        eZHTTPTool::headerVariable('X-PHP-Response-Code', $httpResponseCode, true, $httpResponseCode );
```
Comment (dpobel) [inline lib/ezutils/classes/ezhttptool.php]: nitpick CS: missing space after the opening parenthesis

**Comment by dpobel** [comment]
+1 (besides the CS nitpick)

**Comment by blankse** [comment]
@andrerom  I only have a problem with the login redirect which I solved in a other PR (https://github.com/ezsystems/ezpublish-kernel/pull/1118). I make the changes because the description in JIRA.

@dpobel I corrected the nitpick and rebased to master.

**Comment by andrerom** [comment]
Ok, so if that is the case this is not relevant anymore then. Legacy is in "mature" mode, so we don't merge big changes unless we have to..

**Comment by blankse** [comment]
OK, then I close it.

**Comment by andrerom** [comment]
Thanks @blankse, and sorry for being such a strict maintainer ;)

**Comment by blankse** [comment]
No problem. That sounds good :)

**Pull request closed without merging**